impl/src/main/java/org/ehcache/impl/internal/store/heap/OnHeapStore.java

```diff
@@ -1465,8 +1465,7 @@ protected long byteSized() {
   }
 
   @FindbugsSuppressWarnings("QF_QUESTIONABLE_FOR_LOOP")
-  protected void enforceCapacity() {
-    StoreEventSink<K, V> eventSink = storeEventDispatcher.eventSink();
```

Comment:
This intentionally creates a new event sink.  I can understand how a bunch of code around this area could be reworked such that this wasn't necessary, but right now it is.

Reply:
Yes. It shouldn't be in this PR. It is a commit mistake. Will remove.